### ISSUE TITLE
Small window

### ISSUE THREAD

**Issue by trimechee** [o7q/MediaDownloader#6]
Hello, a user make this surprising review comment in videohelp site ! a lot of softwares has a small window, it's normal, but apparently some users can't see options and features in small windows, maybe it will be good to add option to maximize the size of main window, thank you !

"So tiny! I can hardly read anything on the main display. I can't figure out how to enlarge it. None of the normal methods seem to work. Believe it or not, the only option I've been able to read dealing with size is to Minimize. It's impossible to do anything on a program of that size."

https://www.videohelp.com/software/MediaDownloader/reviews#18434

**Comment by o7q** [comment]
I have been getting a lot of requests about this. Because of how the Visual Studio form designer works it would be difficult to scale up everything without the need to redesign the window. I will look into ways to do it. Currently this problem is not a high priority.

**Comment by trimechee** [comment]
Perfect !  thank you :)

**Comment by trimechee** [comment]
Other user seems to love tiny window : " I have been an avid user of MediaDownloader for as long as it has been out. Since 1.0 I have found that the program is extremely helpful and easy to use. As time has passed, MediaDownloader evolved, and so have I. I've used MediaDownloader for a few things such as: making gifs, downloading sound effects, downloading music, and downloading specific segments of a video. The small compact design of Mediadownloader makes it extremely simple to use and extremely pleasing to the eyes. As a person who gets easily frustrated with having too many things open at once, I can rest safely knowing MediaDownloader will always sit in the corner of my screen."


but another user say : "MediaDownloader no longer works beyond version 3.4.2, Windows 11 Pro 22h2"

https://www.videohelp.com/software/MediaDownloader/reviews#18553
